README.md

```diff
@@ -73,6 +73,22 @@ Currently only a 64-bit version is available.
 The Linux version does not currently automatically update so you will need to
 repeat these steps to upgrade to future releases.
 
+### Archive extraction
+
+An archive is available for people who don't want to install `atom` as root.
+
+This version enables you to install multiple atom versions in parallel. It has been built on Ubuntu 64-bits,
```

Comment:
`atom` should be capitalized, and `64-bits` should be changed to `64-bit` to be consistent with the rest of the documentation.